packages/datagateway-common/src/detailsPanels/dls/visitDetailsPanel.component.tsx

```diff
@@ -282,6 +295,7 @@ const VisitDetailsPanel = (
                 return (
                   <Grid key={sample.id} item xs>
                     <Typography>
+                      {sample.type?.name ? `${sample.type.name}: ` : ''}
                       <b>{sample.name}</b>
                     </Typography>
                   </Grid>
```

Comment:
It might be good to refactor this to only return a Grid and Typography element if we have sample.type.name, instead of returning an empty Typography element in a Grid. So basically refactor to something like:
```
investigationData.samples.filter((sample) => !!sample.type?.name).map((sample) => {
    return (
        <Grid key={sample.id} item xs>
            <Typography>
                {sample.type?.name ? `${sample.type.name}: ` : ''}
                <b>{sample.name}</b>
            </Typography>
        </Grid>
    );
})
```

Reply:
I think it's fine as it is because `sample.name` is required and is always shown, so `Typography` should never be empty. If we filtered samples with no type names, then even their names will be invisible.